src/utils/regex.js

```diff
@@ -1,22 +1,18 @@
 let _ = require('lodash/fp')
 
-let toSafeRegex = _.flow(
-  _.replace(/[.?+*|{}[\]()]/g, '\\$&'),
```

Comment:
@daedalus28 I'm not sure what this was for -- will restore if it's needed -- as you can see I've just turned the junk into the null string instead